crates/proto/src/domain/accounts.rs

```diff
@@ -8,10 +8,12 @@ use miden_objects::{
     Digest,
 };
 
+use super::try_convert;
 use crate::{
     errors::{ConversionError, MissingFieldHelper},
     generated::{
         account as proto,
+        requests::get_account_proofs_request,
         responses::{AccountBlockInputRecord, AccountTransactionInputRecord},
     },
```

Comment:
To follow the convention introduced in #610, I would change this to:
```rust
use crate::{
    errors::{ConversionError, MissingFieldHelper},
    generated as proto,
};
```
And update the rest of the file accordingly.

Reply:
Agreed, changed